orchagent/aclorch.cpp

```diff
@@ -493,17 +491,112 @@ bool AclRuleL3::validateAddAction(string attr_name, string _attr_value)
     {
         value.aclaction.parameter.s32 = SAI_PACKET_ACTION_DROP;
     }
+    else if (attr_value.find(PACKET_ACTION_REDIRECT) != string::npos)
```

Comment:
use tokenize function in common? so that to avoid find and resize functions here?